src/pages/FriendsActivitiesPage.js

```diff
@@ -30,10 +44,28 @@ export default function FriendsActivitiesPage({
         </ArrowBackButton>
       </Header>
       <Main>
+        {categoryTagsAndAll.length > 2 ? (
```

Comment:
Is this intended to show only when 3 or more categories exist?

Reply:
right, as it doesnt make sense to filter when all activities have the same category. And with even only one category, also 'all' appears, which would have exactly the same content as the one category. Therefore I thought it just makes sense to have the filter when you have at least two categories plus all. Or is it not so user friendly? What are your thoughts?😊